client/jetpack-onboarding/steps/contact-form.jsx

```diff
@@ -21,9 +21,13 @@ import { saveJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions'
 
 class JetpackOnboardingContactFormStep extends React.PureComponent {
 	handleAddContactForm = () => {
-		const { siteId } = this.props;
+		const { jpUser, siteId, token } = this.props;
 
-		this.props.recordTracksEvent( 'calypso_jpo_contact_form_clicked' );
+		this.props.recordTracksEvent( 'calypso_jpo_contact_form_clicked', {
+			blog_id: token,
+			site_id_type: 'jpo',
+			user_id_type: jpUser,
```

Comment:
This is the user's email - do you think we can figure a more fitting key name for this property?

Reply:
yes definitely, this was a 'proof of concept' / point for discussion.

Reply:
yes, definitely 👍 This was meant as a point of refs for a discussion.